### PR TITLE
Simplified logging

### PR DESCRIPTION
**Pull request by ejholmes**
Closes https://github.com/remind101/stacker/issues/527

This removes the "loop logger", and replaces it with a simple sequential logger. I think this solves all of the problems talked about in https://github.com/remind101/stacker/issues/527:

1. This will work with parallelized execution.
2. You get color output with `--interactive`.
3. It removes the superfluous "pending" status.
4. It scales easily with large stack configs.

Here's an example of what the output looks like:

[asciicast](https://asciinema.org/a/LgL7DlATfXepyO6XaSVwITcgg)

### CODE REVIEW

**Comment by russellballestrini** [comment]
Wow, this is refreshing, nice work!

File: stacker/logger/__init__.py [hunk 1]
```diff
@@ -1,46 +1,35 @@
...
-def setup_logging(verbosity, interactive=False, tail=False):
-    enable_loop_logger = (
-        verbosity == 0 and
-        sys.stdout.isatty() and
-        not (interactive or tail)
-    )
+def setup_logging(verbosity):
     log_level = logging.INFO
     log_format = INFO_FORMAT
+    if sys.stdout.isatty():
```
Comment (phobologic) [inline stacker/logger/__init__.py]: love this, thanks.

**Comment by phobologic** [comment]
Ok, pushed a small update.  @ejholmes  can you take a look at why the test_util.py tests are all super noisy in tests now?  See: https://circleci.com/gh/remind101/stacker/1404?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

(I thought it was something I had done, but it looks like it's been happening since this update)

**Comment by ejholmes** [comment]
Tests were noisy because the root logger in tests was being switched out with a StreamLogger in `stacker/tests/test_stacker.py`.

I added a commit to this that dependency injects the `setup_logging` function from `scripts/stacker` so unit tests can't fuck with logging.

**Pull request merged**